truss/contexts/image_builder/serving_image_builder.py

```diff
@@ -86,7 +86,9 @@ def __init__(self, truss_dir: Path) -> None:
     def default_tag(self):
         return f"{self._spec.model_framework_name}-model:latest"
 
-    def prepare_image_build_dir(self, build_dir: Optional[Path] = None):
+    def prepare_image_build_dir(
+        self, build_dir: Optional[Path] = None, is_trusted: bool = False
```

Comment:
lets name this param to `use_hf_secret` and default it to false. I think it's more clear for what's happening in the Dockerfile